theme/css/swiper.css

```diff
@@ -0,0 +1,50 @@
+/*testimonial */
+.testimonial-container.swiper-container {
+  width: 100%;
+  height: 450px;
+  padding-top: 10px;
+}
+
+.testimonial-container.swiper-container.full-width {
+  width: 100vw;
+  margin-left: -50vw;
+  margin-right: -50vw;
+}
+
+.testimonial-container.swiper-container .swiper-wrapper {
```

Comment:
You can get rid of this, its not doing anything.

Reply:
done